express-post-json/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+
+const grades = {};
+let id = 3;
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
```

Comment:
Here you should be sending back the grades as an array like in the previous exercises.

Reply:
![Screen Shot 2022-02-09 at 12 14 10 PM](https://user-images.githubusercontent.com/90858352/153263895-dd5ef487-9e6d-4a03-a56c-c6c70d004a65.png)